tier1/{{cookiecutter.project_slug}}/repolinter.json

```diff
@@ -0,0 +1,763 @@
+{
+    "$schema": "https://raw.githubusercontent.com/todogroup/repolinter/master/rulesets/schema.json",
+    "version": 2,
+    "axioms": {
+      "linguist": "language",
+      "licensee": "license",
+      "packagers": "packager"
+    },
+    "rules": {
+      "license-file-exists": {
+        "level": "error",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}LICENSE*"
+            ],
+            "nocase": true
+          }
+        }
+      },
+      "security-file-exists": {
+        "level": "error",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}SECURITY.md"
```

Comment:
@decause-gov Do we want all the markdown files to live in the root directory? Currently, the file location checks are:
- README must be in the root dir
- The rest of the files can live in root or .github/ docs/ subdirs

Reply:
Though I prefer /, I reckon it is acceptable if the .md files live in /docs directory as well, since that is the default location for github-pages hosted sites too.